packages/cli/lib/projects.js

```diff
@@ -168,14 +168,51 @@ const validateProjectConfig = (projectConfig, projectDir) => {
   }
 };
 
+const pollFetchProject = async (accountId, projectName) => {
+  // Temporary solution for gating slowness. Retry on 403 statusCode
+  try {
+    const project = await fetchProject(accountId, projectName);
+    return project;
+  } catch (error) {
+    if (error.statusCode === 404) {
+      throw error;
+    }
+  }
```

Comment:
Try first for all other usages of `ensureProjectExists`, only poll if response is not successful or a 404